components/src/core/components/Grid/grid-utility.scss

```diff
@@ -0,0 +1,16 @@
+@import '../../../styles';
+@import 'variables';
+
+@for $i from 1 through 12 {
```

Comment:
Is it fine to keep a static 12 col grid?

Reply:
12 columns are in [bootstrap 4](https://getbootstrap.com/docs/4.0/layout/grid/#grid-options) and [oxd-spec](https://oxd-prod-infinity.orangehrm.com/main-catalog/grid.html) grid systems. In our application in largest screen size `XL` has only max 4 columns so we should be safe with only 12 columns.